react/src/translations/translation.en.json

```
@@ -291,7 +291,8 @@
       "notifications": "Notifications",
       "signIn": "Sign In",
       "credit": "Credit",
-      "licences": "Licences"
+      "licences": "Licences",
+      "privacy": "Privacy (Korean only)"
```

생각해보니 번역본도 만들어야 할 거 같은데 우선 이렇게 해두었습니다.

---

```suggestion
      "privacy": "Privacy"
```
Footer에 korean only까지 다 들어가면 너무 길어지는거같아서 일단 여기서는 빼는 게 좋을 것 같아요. 추후에 디자인적으로 다른 곳에 korean only를 표시해주거나 영어로 제공할 수 있는지 고민해보겠습니다